tests/src/actionContainers/SwiftActionContainerTests.scala

```diff
@@ -196,7 +198,29 @@ class SwiftActionContainerTests extends BasicActionRunnerTests with WskActorSyst
         })
     }
 
-        it should "properly use KituraNet and Dispatch" in {
+    it should "support pre-compiled binary in a zip file" in {
```

Comment:
@bjustin-ibm do you have an example?
the build script doesn't support this either, right @paulcastro?
if so, we'd need some one off zip file that does this, ... thoughts?